portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -67,11 +73,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String name = request.getParameter("name");
     String comment = request.getParameter("comment");
     String emoji = request.getParameter("emoji");
+    String email = "";
+    UserService userService = UserServiceFactory.getUserService();
+    if (userService.isUserLoggedIn()) {
+      email = userService.getCurrentUser().getEmail();
```

Comment:
Why does it need my email address if I'm logged in? Does this get shared on my comment? It might be worth considering privacy and security even for this weekly project.

Reply:
It gets displayed with your comment right now (it was in the instructions)